### PR TITLE
Enhancement/149/wrap up branch report

### PR DESCRIPTION
**Pull request by eenwemasor**
Add two branch queries to graphql schema

GetBranchReport
GetBranchContributionReport

which returns branch report fields as stated here #149

### CODE REVIEW

File: app/Models/ContributionPlan.php [hunk 1]
```diff
@@ -132,4 +132,13 @@ public function getIsCompletedAttribute()
     {
         return $this->contributionStatus === self::STATUS_COMPLETED;
     }
+
+
+    public function getContributionInterest()
```
Comment (omadoyeabraham) [inline app/Models/ContributionPlan.php:137]: @enwemasor1995  Please rename p, i and t with proper variable names that explain what they are

Comment (eenwemasor) [reply app/Models/ContributionPlan.php:137]: @omadoyeabraham 
variables renamed

File: app/Services/BranchService.php [hunk 2]
```diff
@@ -152,4 +159,173 @@ public function getBranchStatistics(string $branch_id): array
...
         return $statistics;
     }
+
+    /**
+     * @param $branch_id
+     * @param null $start_date
+     * @param null $end_date
+     * @return array
+     */
+    public function getBranchReport($branch_id, $start_date = null, $end_date = null)
```
Comment (omadoyeabraham) [inline app/Services/BranchService.php:169]: @enwemasor1995 I said you should remove all DB query related code from the service into the repository

**Review by omadoyeabraham — APPROVED** [review]

File: app/Services/BranchService.php (hunk 2 above)
Comment (eenwemasor) [reply app/Services/BranchService.php:169]: I did no add this function

**Pull request merged**